frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -313,92 +318,96 @@ export function AnsibleAutomationsForm(props: {
-                                },
-                                onCreate: () => {
-                                    setEditAnsibleJobs({ jobs: scalePostJobs, setJobs: setScalePostJobs })
-                                    setEditAnsibleJob({ name: '', extra_vars: {} })
-                                },
-                            },
-                        ],
-                    },
-                    {
-                        type: 'Section',
-                        title: t('template.create.destroy'),
-                        wizardTitle: t('template.create.destroy.wizard.title'),
-                        inputs: [
-                            {
-                                id: 'destroyPreJob',
-                                type: 'OrderedItems',
-                                label: t('template.preDestroy.label'),
-                                placeholder: t('template.job.placeholder'),
-                                value: destroyPreJobs,
-                                onChange: setDestroyPreJobs,
-                                keyFn: (ansibleJob: AnsibleJob) => ansibleJob.name,
-                                cellsFn,
-                                onEdit: (ansibleJob) => {
-                                    setEditAnsibleJobs({ jobs: destroyPreJobs, setJobs: setDestroyPreJobs })
-                                    setEditAnsibleJob(ansibleJob)
-                                },
-                                onCreate: () => {
-                                    setEditAnsibleJobs({ jobs: destroyPreJobs, setJobs: setDestroyPreJobs })
-                                    setEditAnsibleJob({ name: '', extra_vars: {} })
-                                },
-                            },
-                            {
-                                id: 'destroyPostJob',
-                                type: 'OrderedItems',
-                                label: t('template.postDestroy.label'),
-                                placeholder: t('template.job.placeholder'),
-                                value: destroyPostJobs,
-                                onChange: setDestroyPostJobs,
-                                keyFn: (ansibleJob: AnsibleJob) => ansibleJob.name,
-                                cellsFn,
-                                onEdit: (ansibleJob) => {
-                                    setEditAnsibleJobs({ jobs: destroyPostJobs, setJobs: setDestroyPostJobs })
-                                    setEditAnsibleJob(ansibleJob)
-                                },
-                                onCreate: () => {
-                                    setEditAnsibleJobs({ jobs: destroyPostJobs, setJobs: setDestroyPostJobs })
-                                    setEditAnsibleJob({ name: '', extra_vars: {} })
-                                },
-                            },
-                        ],
-                    },
+                    ...(automationFeatureGateFlag
```

Comment:
Can `&&` work instead of doing a ternary?

Reply:
i think the `...` operator does not respond well to the case `...(undefined)` so we need to be able to return an empty array.